File: Gateway/src/main/java/com/kbds/gateway/utils/DateUtils.java
package com.kbds.gateway.utils;

import java.text.SimpleDateFormat;
import java.util.Date;

/**
 * 
 *
 * <pre>
 *  Class Name     : DateUtils.java
 *  Description    : 날짜 관련 유틸 클래스
 *  Author         : 구경태 (kyungtae.koo@kbfg.com)
 * 
 * -------------------------------------------------------------------------------
 *     변경No        변경일자        	       변경자          Description
 * -------------------------------------------------------------------------------
 *     Ver 1.0      2020-06-16    	   구경태          Initialized
 * -------------------------------------------------------------------------------
 * </pre>
 *
 */
public class DateUtils {

  /**
   * 현재시간 YYYY-MM-DD HH:MM:SS로 변환
   * @return  날짜 문자열
   */
  public static String getCurrentTime() {

    return new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(new Date());
  }
}

File: GatewayLog/src/main/kotlin/com/kbds/gatewaylog/utils/DateUtils.kt
package com.kbds.gatewaylog.utils

import java.text.ParseException
import java.text.SimpleDateFormat
import java.util.*

/**
 * <pre>
 *  Class Name     : DateUtils.kt
 *  Description    : 날짜 Utility 기능
 *  Author         : 구경태 (kyungtae.koo@kbfg.com)
 *
 * -------------------------------------------------------------------------------
 *     변경No        변경일자                변경자          Description
 * -------------------------------------------------------------------------------
 *     Ver 1.0      2020-04-20     		   구경태          Initialized
 * -------------------------------------------------------------------------------
 * </pre>
 */
class DateUtils {

    companion object {

        /**
         * 날짜 타입 유효성 체크 메소드
         */
        @Throws(ParseException::class)
        fun validateDateFormat(vararg dates: String) {

            for (date in dates) {

                var formatter = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.getDefault())
                formatter.parse(date)
            }
        }
    }
}